app/src/main/java/com/example/poste/api/poste/models/Folder.java

```diff
@@ -92,6 +92,15 @@ public void setPosts(List<Post> posts) {
         this.posts = posts;
     }
 
+    /**
+     * Removes a post from the list of posts in this folder
+     *
+     * @param post the post to remove
+     */
+    public void removePost(Post post) {
+        posts.remove(post);
```

Comment:
See comments above re: API call for this functionality to avoid client-side / backend sync issues

Reply:
Discussed below.